osbenchmark/worker_coordinator/worker_coordinator.py

```
@@ -963,44 +976,55 @@ def create_os_clients(self):
+                ]
+            else:
+                # For non-OpenSearch databases, skip OS-specific telemetry devices
+                self.logger.info("Skipping OpenSearch-specific telemetry devices for database type [%s]", database_type)
+                devices = []
         else:
             devices = []
         self.telemetry = telemetry.Telemetry(enabled_devices, devices=devices)
 
-    def wait_for_rest_api(self, opensearch):
-        os_default = opensearch["default"]
+    def wait_for_rest_api(self, database):
```

Ruff (0.14.13)

Unused method argument: `database`

(ARG002)